core/tests/test_utils.py

```diff
@@ -2383,8 +2373,11 @@ def _send_post_request(
         # Convert the files to bytes.
         if upload_files is not None:
             upload_files = tuple(
-                tuple(python_utils.convert_to_bytes(f) for f in upload_file)
-                for upload_file in upload_files)
+                tuple(
+                    f.encode() if isinstance(f, str) else f
```

Comment:
Shouldn't we usually specify an explicit encoding when we call encode() (here and elsewhere)?

Reply:
Right, done-